qiskit/algorithms/optimizers/spsa.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2018, 2021.
```

Comment:
why change this?

Reply:
because I've overwritten the whole file with the new implementation 😉 I'll revert to the old copyright years